packages/@glimmer-workspace/integration-node-tests/test/prettier.test.ts

```diff
@@ -0,0 +1,207 @@
+import { mkdtemp, readFile, writeFile } from 'node:fs/promises';
+import { createRequire } from 'node:module';
+import os from 'node:os';
+import { join } from 'node:path';
+
+import { $ } from 'execa';
+import * as prettier from 'prettier';
+import { describe, expect, it } from 'vitest';
+
+const monorepoRoot = join(import.meta.dirname, '../../../../');
+const require = createRequire(import.meta.url);
+
+/**
+ * See: https://github.com/glimmerjs/glimmer-vm/issues/1688
+ *
+ * Requires the root package.json#pnpm#overrides point at our internal
+ * copy of @glimmer/syntax, or else prettier brings its own already published
+ * copy of @glimmer/syntax
+ */
+describe('Prettier', () => {
+  it(`SANITY: we've symlinked to the in-repo copy of @glimmer/syntax`, () => {
```

Comment:
I try to use the term "SMOKE" rather than "SANITY" in general.

Reply:
updated!